helm/templates/backend/backend.serviceaccount.yaml

```diff
@@ -35,6 +35,9 @@ rules:
 - apiGroups: [""]
   resources: ["secrets"]
   verbs: ["create", "delete"]
+- apiGroups: ["networking.k8s.io"]
+  resources: ["ingresses"]
+  verbs: ["create", "delete"]
```

Comment:
Didn't try it yet locally or in the cluster but you have to also distinguish the openshift/ non openshift case here. So just like in our routing templates you can do the following:
```
{{- if eq .Values.cluster.kind "Kubernetes" }}
- apiGroups: ["networking.k8s.io"]
  resources: ["ingresses"]
  verbs: ["create", "delete"]
{{- else }}
- apiGroups: ["route.openshift.io/v1"]
  resources: ["route"]
  verbs: ["create", "delete"]
{{- end }}
```

Reply:
Nice find!